enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -191,4 +191,18 @@ impl<'a> Handler<'a> {
             ret
         }
     }
+
+    /// rt_sigaction() syscall stub
+    /// This type of signal is not delivered to the enclave, so a dummy value
```

Comment:
Hmm, the comment is kind of wrong.

Even though, the outside world does not send a signal to the enclave process, the enclave could use `kill()` to send itsself a signal.

We also have all other kind of signal, see e.g. the `si_code` field documentation in the `rt_sigaction` man page.